docker-compose.dev.yml

```diff
@@ -0,0 +1,34 @@
+version: '3.4'
+
+services:
```

Comment:
Is NATS an integral part of the IAM Org creator or it could use any NATS cluster?
If the latter then I am missing the "not-docker-compose" command to run it. I forsee iam-org-creator to expect nats cluster url in some kind env variable that i can pass during the startup. 

In kubernetees I am simply taking the image, and deploy & run it passing all of the needed env. variables. Can you thins kind of line to the readme file?

Addtionally, should the NATS cluster be also deployed to k8s along with iam-org-creator?

Reply:
@KaamilW NATS is not part of the iam-org-creator deployment. So it should not be deployed to k8s along with iam-org-creator. I think it is only included in this docker-compose for testing purposes.

As for a running standalone from env variables, I think it would be something like `docker run iam-org-create --env-file .env`. @ahmedolaibrahim could you help prep the `.env` files that Kamil will need for both the `dev` and `volta` environments? Let me know if you have any questions

Reply:
Thanks @jrhender .
@ahmedolaibrahim  would you please add to readme.md example line of how to run the app from the docker image (either including needed  env variables or with usage of .env file?

Reply:
so @KaamilW to run the docker app, all we need to do is run `docker-compose up` which is already stated in the readme docs, to know which env variables  the docker container is to use,  that is added in this [file](https://github.com/energywebfoundation/iam-org-creator/blob/ee9ca8c0eb352a7cfc25e5ad71798864c518f6ca/docker-compose.dev.yml#L16) which is what is needed for local dev setup. I will provide the env.prod file for production.

Reply:
Hey @ahmedolaibrahim  -> docker-compose set-up is different thank k8s one where pure image is being used. Nevertheless, as proposed lets discuss it tomorrow to clarify doubts.